web/config.py

```diff
@@ -761,6 +761,10 @@
     }
 ]
 
+# The claim which is used for the username. If the value is empty the
+# email is used as username, but if a value is provided, the claim has to exist.
+OAUTH2_USERNAME_CLAIM: None
```

Comment:
This should be `OAUTH2_USERNAME_CLAIM = None`

Reply:
Yes, you're right. I've changed it. Thank you!